tests/testthat/test-infrastructure.r

```diff
@@ -59,3 +59,68 @@ test_that("use_data", {
     expect_equal(local_test_data_item_to_save, 42L)
   })
 })
+
+test_that("use_package", {
```

Comment:
This test feels like overkill to me - the function is not that complicated so I don't think it needs an extensive testing harness. I'd rather have a few smaller tests that each tested on specific failure point.